core/domain/question_domain.py

```diff
@@ -307,8 +307,8 @@ def to_dict(self):
             'id': self.id,
             'creator_id': self.creator_id,
             'question_content': self.question_content,
-            'last_updated': self.last_updated,
-            'created_on': self.created_on
+            'last_updated': utils.get_time_in_millisecs(self.last_updated),
```

Comment:
Hm. Probably should be last_updated_msec, created_on_msec then? In general, try not to use the same name for different things.

Reply:
Done.